SU-HSDO/stanford_mrc
# MRCD8-317 speaker field

# READY FOR REVIEW

# Summary
- Updated template files to mirror one another for consistency
- Updated styles for spacing and sizing

# Needed By (Date)
- End of sprint (3/29)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Make sure that the templates follow this model (as available in the content): Program, Date, Time and Location, Speaker, and Title
3. Make sure spacing and sizing is consistent (Featured Events Card should have larger font size for fields other than title)

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-317

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
